pkgs/development/python-modules/neovim_gui/default.nix

```diff
@@ -20,9 +21,16 @@ buildPythonPackage rec {
     sha256 = "1vpvr3zm3f9sxg1z1cl7f7gi8v1xksjdvxj62qnw65aqj3zqxnkz";
   };
 
-  propagatedBuildInputs = [ neovim click pygobject3 pkgs.gobject-introspection pkgs.makeWrapper pkgs.gtk3 ];
+  propagatedBuildInputs = [ pynvim click pygobject3 pkgs.gobject-introspection pkgs.makeWrapper pkgs.gtk3 ];
 
-  patchPhase = ''
+  patches = [
+    (fetchpatch {
+      url = https://github.com/neovim/python-gui/commit/48b714c96f0d0ea46921a1b670b2eec851e7af00.patch;
+      sha256 = "1gia7ksyf8x7lkawrslq3ppv6k0hpp87q19g3hi28fl7lhh17kjn";
+    })
+  ];
+
```

Comment:
seems like your patch was accepted upstream. Can we bump the gui instead of patching it ?

Reply:
They didn't release a new version, so bumping gui requires pinning to a specific commit and switching to "unstable" version.

Given the project is abandoned I wonder if anyone uses it at all? (i.e., do we still need to package it, or it is safe to drop?)

Reply:
I think we can drop it.

Reply:
the python neovim gui is mostly the experimental GUI where neovim devs (mostly bjorn) make sure the UI features actually work in a sensible way. It's not supposed to be used really so it can be dropped. I might save the recipe into my overlay though.

Reply:
Thanks for the info! I've dropped it.